### DISCUSSION TITLE
Can't buy a domain on Vercel

### DISCUSSION THREAD

**Discussion by merryface** [vercel/community#308] [Help] [answered]
Trying to buy a domain on vercel, but it keeps failing for some reason. The payment goes through, but then the app says "An unexpected error occurred (registration_failed). Please try again later."

The payment is then immediately reversed. I tried changing browsers but that didn't help.

Anyone else experience this? Seems like they can't register the domain name for some reason.

**Comment by gogocharli** [comment, marked as answer]
Hello @merryface, what is the TLD of the domain you are trying to purchase on Vercel?

**Reply by samcx** [reply]
Marking this as solved until we hear back!
